### PR TITLE
Update addons.md

### PR DESCRIPTION
**Pull request by lucasGithuber**
now linking to right page of builds

### CODE REVIEW

**Comment by FN-FAL113** [comment]
why make it specific to build #8?

**Comment by svr333** [comment]
yup its pointing to a specific build which isnt good

File: pages/Addons.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ PotionExpansion | EpicPlayerA10 | <details><summary>expand</summary>An addon whi
...
-ElementManipulation | lucasGithuber | <details><summary>expand</summary>A slimefun addon that adds periodic table elements and more machines, giving the player the capacity to create items using these elements.</details> | [Readme](https://github.com/lucasGithuber/Element-Manipulation/blob/master/README.md) | [GitHub](https://github.com/lucasGithuber/Element-Manipulation) | [Releases](https://github.com/lucasGithuber/Element-Manipulation/releases)
+ElementManipulation | lucasGithuber | <details><summary>expand</summary>A slimefun addon that adds periodic table elements and more machines, giving the player the capacity to create items using these elements.</details> | [Readme](https://github.com/lucasGithuber/Element-Manipulation/blob/master/README.md) | [GitHub](https://github.com/lucasGithuber/Element-Manipulation) | [Builds](https://thebusybiscuit.github.io/builds/lucasGithuber/Element-Manipulation/master/#8)
```
Comment (Sefiraat) [inline pages/Addons.md:69]:
```suggestion
ElementManipulation | lucasGithuber | <details><summary>expand</summary>A slimefun addon that adds periodic table elements and more machines, giving the player the capacity to create items using these elements.</details> | [Readme](https://github.com/lucasGithuber/Element-Manipulation/blob/master/README.md) | [GitHub](https://github.com/lucasGithuber/Element-Manipulation) | [Builds](https://thebusybiscuit.github.io/builds/lucasGithuber/Element-Manipulation/master/)
```

**Review by Sefiraat — DISMISSED** [review]

File: pages/Addons.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ PotionExpansion | EpicPlayerA10 | <details><summary>expand</summary>An addon whi
...
-ElementManipulation | lucasGithuber | <details><summary>expand</summary>A slimefun addon that adds periodic table elements and more machines, giving the player the capacity to create items using these elements.</details> | [Readme](https://github.com/lucasGithuber/Element-Manipulation/blob/master/README.md) | [GitHub](https://github.com/lucasGithuber/Element-Manipulation) | [Releases](https://github.com/lucasGithuber/Element-Manipulation/releases)
+ElementManipulation | lucasGithuber | <details><summary>expand</summary>A slimefun addon that adds periodic table elements and more machines, giving the player the capacity to create items using these elements.</details> | [Readme](https://github.com/lucasGithuber/Element-Manipulation/blob/master/README.md) | [GitHub](https://github.com/lucasGithuber/Element-Manipulation) | [Builds](https://thebusybiscuit.github.io/builds/lucasGithuber/Element-Manipulation/master/)
```
Comment (Sefiraat) [inline pages/Addons.md:69]:
```suggestion
ElementManipulation | lucasGithuber | <details><summary>expand</summary>A Slimefun addon that adds periodic table elements and more machines, giving the player the capacity to create items using these elements.</details> | [Readme](https://github.com/lucasGithuber/Element-Manipulation/blob/master/README.md) | [GitHub](https://github.com/lucasGithuber/Element-Manipulation) | [Builds](https://thebusybiscuit.github.io/builds/lucasGithuber/Element-Manipulation/master/)
```

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**